src/frequenz/sdk/timeseries/battery_pool/_battery_pool_reference_store.py

```diff
@@ -21,8 +21,15 @@
 from ._methods import MetricAggregator
 
 
-class BatteryPool:  # pylint: disable=too-many-instance-attributes
-    """A class for maintaining the state/tasks for a set of pool of batteries.
+class BatteryPoolReferenceStore:  # pylint: disable=too-many-instance-attributes
+    """A class for maintaining the shared state/tasks for a set of pool of batteries.
+
+    This includes ownership of
+    - the formula engine pool and metric calculators.
+    - the tasks for updating the battery status for the metric calculators.
+
+    These are independent of the priority of the actors, and can be shared between
```

Comment:
The `,` is [wrong](https://www.grammarly.com/blog/comma-before-and/) as the second part of the conjunction is not an independent clause.